working-groups.md

```diff
@@ -0,0 +1,65 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
+
+📅 Weekly calls  
+🌱 Cultivators:   
```

Comment:
This seems unnecessary as it changes all the time

Reply:
I'm agnostic on this, was on loomio but yes, we could remove

Reply:
agree with ben. would prefer to have all these roles in a single org-wide spreadsheet, maybe with monthly granularity (publiclab.org does something like this for chairing/running projects)

Reply:
removing